src/Reflection/Adapter/ReflectionObject.php

```diff
@@ -222,15 +222,15 @@ public function getConstants(?int $filter = null): array
      */
     public function getConstant($name)
     {
-        return $this->betterReflectionObject->getConstant($name);
+        return $this->betterReflectionObject->getConstantValue($name);
```

Comment:
Do we even need `getConstantValue` in our API? Can't a `ReflectionConstant` provide that, when asked for it?